ada/language/ast.py

```diff
@@ -7236,7 +7247,7 @@ def is_static_call():
         .. note:: This is an experimental feature. There might be some
             discrepancy with the GNAT concept of "static call".
         """
-        return Entity.is_call & Not(Entity.is_dispatching_call)
+        return Entity.is_direct_call & Not(Entity.is_dispatching_call)
```

Comment:
If what I said above is correct, this should make this redundant. Also please put those changes in a subsequent commit :P